app/src/main/java/com/ganaljigi/kubf/ui/home/screen/HomeScreen.kt

```
@@ -110,33 +113,71 @@ fun HomeScreen(
+                val fusedLocationClient = LocationServices.getFusedLocationProviderClient(context)
+                fusedLocationClient.getCurrentLocation(
+                    Priority.PRIORITY_HIGH_ACCURACY,
+                    null
+                ).addOnSuccessListener { location ->
+                    location?.let {
+                        viewModel.updateUserLocation(
+                            LatLng(it.latitude, it.longitude)
+                        )
+                    }
+                }
+            } catch (e: SecurityException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)